documentation/book/cluster-operator.adoc

```diff
@@ -78,6 +78,26 @@ A JSON string representing the storage configuration for the Kafka broker nodes.
 `kafka-metrics-config`::
 A JSON string representing the JMX exporter configuration for exposing metrics from Kafka broker nodes.
 When this field is absent no metrics will be exposed.
+`kafka-logging`::
+A JSON string representing the level of logging for Kafka nodes. This field can contain JSON with loggers, which should be changed and levels.
+Or there can be specified name of config map, which should be used for setting logger levels.
+When this field is absent default values are used.
```

Comment:
I noticed that we list what the default values for other descriptions in the documentation. Should we list the default values here as well? Or would the default JSON string be too large?

Reply:
I think you should document that there is _a_ default. If you can succinctly describe (in words) what that default is, then you should do so. I don't think it is really necessary to document the exact default.